platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/cli/EventStreamInfoCommand.java

```diff
@@ -48,7 +55,20 @@ private void setEventStreamDirectory(final Path eventStreamDirectory) {
             names = {"-f", "--first-round"},
             description = "The first to be considered.")
     private void setFirstRound(final long firstRound) {
-        this.firstRound = firstRound;
+        boundBuilder.setRound(firstRound);
+    }
+
+    @CommandLine.Option(
+            names = {"-t", "--timestamp"},
+            description = "The the timestamp to be considered.")
```

Comment:
"the the"

I'd be specific and say that this is is a lower bound. Not obvious based on this sentence alone that you want to start looking at the stream at this timestamp. Perhaps something like "The minimum timestamp to be considered".

I'd describe the format here. Ideally somebody could figure out how to use this tool without reading the source code.

Reply:
updated